Time Tracker/Button.cs

```diff
@@ -10,19 +10,20 @@ public class Button
         public Button()
         {
             this._stopwatch = new Stopwatch();
-            this._isPressed = false;
         }
 
         public string Name { get; set; }
+        public bool IsPressed { get; set; }
 
         public void Press()
         {
             //First two lines will change
 
             TimeSpan ts = _stopwatch.Elapsed;
             string elapsedTime = String.Format("{0:00}:{1:00}:{2:00}", ts.Hours, ts.Minutes, ts.Seconds);
-            _isPressed = !_isPressed;
-            if (this._isPressed)
+
+            IsPressed = !IsPressed;
```

Comment:
I don't like this line

Reply:
It's almost same as before?

Reply:
Its the same. But I dont like any of them :D